source/modules/collection-api/address/overview.md.erb

```diff
@@ -0,0 +1,26 @@
+## Addresses
+```shell
+{
+    "street_name":   "Rua Fidalga",
+    "street_number": "252",
+    "zip_code":      "05432-010",
+    "city":          "São Paulo",
+    "state":         "SP",
+    "country":       "Brasil",
+    "latitude":      -43.2360,
+    "longitude":     -23.5115
+}
+```
+
+Numerous resources have associated location data: Sale payments have an associated billing address, events occur at a particular address, or an account may have a shipping address associated with it. Inspetor standardizes the format in which we collect this data as follows:
+
+Property          | Required? | Type   | Description
+--------          | --------- | -----  | -----------
+**street**        | Yes       | String | The street of the address
```

Comment:
I think this is supposed to be `street_name` (that's what's being used in the `curl` examples)